persons/personsGraph.json

```diff
@@ -0,0 +1,7656 @@
+{
```

Comment:
I assume you'll need this to generate graphs unless you have this data elsewhere.

Reply:
This is for all the persons records, will need to be periodically generated.